content/700-optimize/300-recordings.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Recordings'
+metaTitle: 'Optimize: Recordings'
+metaDescription: "Learn about using Optimize's recording feature."
+tocDepth: 3
+toc: true
+---
+
+The Recordings feature is designed to help developers debug and isolate different sets of queries into distinct sessions, known as recording. This targeted approach enables precise performance analysis and optimization by preventing the mixing of queries from different applications or test rounds, leading to clearer insights and more effective debugging.
+
+## Managing a recording session
+
+You can manually start and stop recording sessions via the [Optimize dashboard](https://optimize.prisma.io) by clicking the **Start Recording** and **Stop Recording** buttons.
+
+:::warning
+
+A recording will automatically stop if the 10k query limit is reached or if the Prisma schema of the app is changed.
+
+:::
+
+A recording session can be deleted when no longer needed.
+
+## Identifying a recording session
+
+You can rename and tag your recording sessions for easy identification and context. Click on the generated default name of the recording session and type the desired name.
+
+## Data captured in a recording session
+
+Each recording session captures insights about queries executed in your app, including:
+- All executed queries during the session.
+- The raw query generated and sent to the database by Prisma ORM as **Raw Query**.
+- The number of times a query pattern was executed as **Count**.
+- [Query performance metrics](/optimize/performance-metrics).
+- Errors encountered during query execution.
+
+Each recording can have a maximum of 10k queries. There are no limits on the storage retention period.
+
+## Recommendations from a recording session
+
+When a recording session ends, Optimize generates recommendations such as:
+- [Excessive number of rows returned](/optimize/recommendations/excessive-number-of-rows-returned)
+- [Full table scans caused by LIKE operations](/optimize/recommendations/full-table-scans-caused-by-like-operations)
+- [Queries on unindexed columns](/optimize/recommendations/queries-on-unindexed-columns)
+
+:::info
+Use the [AI Explainer](/optimize/ai-explainer) feature to better understand recommendations and apply them within your Prisma model context.
```

Comment:
We can also phrase this to say "Use the AI Explainer to ask questions about the recommendations".